content/news/2023-02-22-news-2023-02-22-new-r-ladies-website/index.en.md

```diff
@@ -0,0 +1,103 @@
+---
+title: New R-Ladies Website
+author: Athanasia M. Mowinckel
+date: '2023-02-22'
+slug: news/2023-02-22-new-r-ladies-website
+categories:
+  - R-Ladies
+tags: []
+---
+
+
+We finally managed to release a new R-Lades website!
+
+It's been a long journey, with many people involved, and we are happy to be able to finally share with you this new site and some new content.
+
+<!--more-->
+
+There were some key issues we wanted to resolve with our webpage, that we could not easily maintain with the Wordpress site we had going:
+
+- Multilingual website 
+  - This is not the easiest to maintain with Wordpress
+- Very slow directory
+  - The database was just too slow and heavy, page loading time was horrible
+- Integrate the blog into the website
+  - The blog was a Blogdown site maintained through Github and Netlify
+- Easier long-term maintenance and collaboration
+  - Wordpress would require making a user for every person that wanted to contribute to the site
+  - Changing to something hosted on GitHub would mean easier community help and collaboration
+
+## History
+The work already started in 2019, with Bea Hernandez, Daloha Rodríguez-Molina and Maëlle Salmon.
+The initial work was in making a [blogdown]() website, which utilises [Hugo]() and Rmarkdown integration.
+It was a natural place to start porting our Wordpress site into something more common to use in our community. 
+Additionally, the Wordpress site was requiring more and more maintenance, and in particular our [R-Ladies directory]() was so slow we were getting reports of people not using it because the loading time of the page was so long!
+
+In 2020, I (Athanasia Mowinckel) was on-boarded to the website team, initially to maintain the Wordpress site while the new Hugo site was being built.
+After just a little while, I also started working on the blogdown site. 
+
+In mid-2020 we decided that the webpage was better served as a pure Hugo site, not as blogdown. 
+At that point, it was due to some Hugo features not existing in Blogdown (yet) that we wanted to utilise for the website.
+These were settings for multi-lingual websites we really wanted to take advantage of.
+Furthermore, since only the blog would ever need code execution, having a site build that does not need R means a much faster build, and easier to maintain.
+
+Then Covid hit, and we all felt the stress of that period.
+Development went slow, and things dragged out.
```

Comment:
When reading this I don't see your huge work. Maybe add a sentence or more about the fact you created the Hugo stuff